lib/asymmetricKeyDetailsSupported.js

```diff
@@ -1,3 +1,4 @@
-const semver = require('semver');
+const [major, minor] = process.version.slice(1).split('.').map(parseInt)
```

Comment:
There's many problems with this line :

1. According to nodejs this couldn't work:
    > The process.version property contains the **Node.js** version string.
 
    You can use [`process.env.npm_package_version`](https://docs.npmjs.com/cli/v6/using-npm/scripts#packagejson-vars) but this presumes everyone is using npm. Or you can use `const { version } = require('../package.json');` to avoid that assumption
2. This also passes element index as radix to parseInt and causes strange behavior :
  `"v9.0.0".slice(1).split('.').map(parseInt) // Array(3) [ 9, NaN, 0 ]`
  fixed by using `.map(v => parseInt(v, 10))`
3. This doesn't take semver extensions into account correctly:
    `"v2.0.0-rc.1".slice(1).split('.').map(v => parseInt(v, 10)) // Array(4) [ 2, 0, 0, 1 ]`
    As you don't need them, you can limit parsing to major, minor : `version.split('.').slice(0, 2).map(v => parseInt(v, 10))`

Reply:
Yes. You are right. First element is value, second is index. But we dont need to care for semver Extensions because we just need major and minor.

```suggestion
const [major, minor] = process.version.slice(1).split('.').map(function (v) { return parseInt(v, 10) })
```